File: eventos-api/app/models/usuario.py
from sqlalchemy import Column, String, Boolean, DateTime
from sqlalchemy.dialects.postgresql import UUID
import uuid
from app.core.database import Base
import datetime

class Usuario(Base):
    __tablename__ = "usuarios"
    
    id = Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
    nome = Column(String, nullable=False)
    email = Column(String, unique=True, nullable=False)
    senha_hash = Column(String, nullable=False)
    cpf = Column(String, unique=True)
    email_verificado = Column(Boolean, default=False)
    papel = Column(String)
    criado_em = Column(DateTime, default=datetime.datetime.utcnow)
    atualizado_em = Column(DateTime, default=datetime.datetime.utcnow, onupdate=datetime.datetime.utcnow)

File: eventos-admin/eventos_admin/src/main/java/com/eventosmanager/models/Usuario.java
package com.eventosmanager.models;

import java.util.UUID;
import java.time.LocalDateTime;

public class Usuario {
    public UUID id;
    public String nome;
    public String email;
    public String cpf;
    public Boolean email_verificado;
    public String papel;
    public LocalDateTime criado_em;
    public LocalDateTime atualizado_em;
}
